docs/accessories/index_accessories.mdx

```diff
@@ -5,17 +5,21 @@ sidebar_position: 1
 ---
 
 <center>
-  <img
-    src="/img/about_images/about_1.jpg"
-    width="1000"
-  />
+  <figure>
+    <a href="https://norlab.ulaval.ca/">
```

Comment:
Are we sure that we want to be linking to third-party pages from here?

Reply:
I will change this to a related norlab blog post on clearpathrobotics.com/blog